packages/fluentui/react-builder/src/components/Canvas.tsx

```diff
@@ -153,6 +222,66 @@ export const Canvas: React.FunctionComponent<CanvasProps> = ({
       const iframeDocument = iframe.contentDocument;
       const iframeWindow = iframe.contentWindow;
 
+      setVirtualCursorElements(
+        Array.from(
+          iframeDocument.querySelectorAll(
+            [
+              'a',
+              'button',
+              'h1',
+              'h2',
+              'h3',
+              'h4',
+              'h5',
+              'h6',
+              'input',
+              // 'li',
+              'marquee',
+              // 'ol',
+              // 'option',
+              // 'p',
+              'select',
+              'textarea',
+              // 'ul',
+              '[role="button"]',
+              '[role="checkbox"]',
+              // '[role="dialog"]',
+              '[role="gridcell"]',
+              '[role="link"]',
+              // '[role="list"]',
+              '[role="listbox"]',
+              '[role="listitem"]',
+              '[role="log"]',
+              // '[role="menu"]',
+              // '[role="menubar"]',
```

Comment:
Are you intentionally leaving the commented roles in the codebase ?

Reply:
Yes, I am. The commented selectors are intentionally not included because they are narrated as landmarks or groups when enterring elements. I will add a comment explaining why they are commented out.

Reply:
I wanted to keep them there in case they will be added in future and so tthat it's clear that these selectors are not missed out but ARE intentionally not included.